src/Items.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+
+export default class Items extends Component {
+    render() {
+     const  jsxItems=[]
+    
+      const itemsList= this.props.items
+      for(let i=0; i<itemsList.length; i++){
+          const li = <li>{itemsList[i].name}</li>
+          jsxItems.push(li)
+      }
```

Comment:
This is great! This is how you build a list of elements in react. You're passing a prop down and looping over them, building an array then rendering that array. Any reason you chose to use a for loop here and not something like `.map()`?